Development/nmos/settings.h

```diff
@@ -270,6 +278,40 @@ namespace nmos
             // system_interval_min/system_interval_max [node]: used to poll for System API changes; default is about one hour
             const web::json::field_as_integer_or system_interval_min{ U("system_interval_min"), 3600 };
             const web::json::field_as_integer_or system_interval_max{ U("system_interval_max"), 3660 };
+
+            // system_label [registry]: used in System API resource label field
+            const web::json::field_as_string_or system_label{ U("system_label"), U("") };
+
+            // system_description [registry]: used in System API resource description field
+            const web::json::field_as_string_or system_description{ U("system_description"), U("") };
+
+            // system_tags [registry]: used in System API resource tags field
+            // "Each tag has a single key, but MAY have multiple values. Each tags SHOULD be interpreted using the comparison of a single key value pair,
+            // with the comparison being case-insensitive following the Unicode Simple Case Folding specification."
+            // {
+            //     "tag_1": [ "tag_1_value_1", "tag_1_value_2" ],
+            //     "tag_2": [ "tag_2_value_1" ]
+            // }
+            // See https://specs.amwa.tv/is-09/releases/v1.0.0/docs/2.1._APIs_-_Common_Keys.html#tags
+            const web::json::field_as_value_or system_tags{ U("system_tags"), web::json::value::object() };
+
+            // "syslog contains hostname and port for the system�s syslog �version 1� server using the UDP transport (IETF RFC 5246)"
```

Comment:
Non-ASCII apostrophe and double quotation marks should be replaced.

Reply:
done